spec/requests/publishers/ats_api/v1/vacancies_spec.rb

```diff
@@ -382,21 +513,220 @@
 
       security [api_key: []]
 
+      parameter name: :vacancy, in: :body, schema: {
+        type: :object,
+        additionalProperties: false,
+        required: %i[external_advert_url
+                     expires_at
+                     job_title
+                     job_advert
+                     skills_and_experience
+                     salary
+                     visa_sponsorship_available
+                     external_reference
+                     is_job_share
+                     job_roles
+                     working_patterns
+                     contract_type
+                     phases
+                     schools],
+        properties: {
+          external_advert_url: { type: :string, example: "https://example.com/jobs/123" },
+          publish_on: { type: :string, format: :date },
+          expires_at: { type: :string, format: :date },
+          job_title: { type: :string, example: "Teacher of Geography" },
+          job_advert: { type: :string, example: "Teacher of Geography" },
```

Comment:
oops typo

Reply:
job_advert example is 'Teacher of Geography' - is this mean to be a URL, or something else?